mmf/datasets/databases/image_database.py

```diff
@@ -185,7 +185,7 @@ def _get_attrs(self, item):
         if pick == "identifier" and "left_url" in item and "right_url" in item:
             return [image + "-img0.jpg", image + "-img1.jpg"]
         elif pick == "image_name" or pick == "image_id":
-            return [image + ".jpg"]
+            return [image + ".jpeg"]
```

Comment:
Can you comment on why this is needed? It might make sense to add unique behavior for mmimdb.

Reply:
The extension for imdb images is jpeg. So I changed it. But I am not sure the cases of other datasets.

Reply:
Cool! Image database will anyways find correct extension later on.